en/simulation/ignition_gazebo.md

```diff
@@ -18,28 +18,28 @@ See [Simulation](../simulation/README.md) for general information about simulato
 ## Installation (Ubuntu Linux)
 
 :::note
-These instructions were tested on Ubuntu 18.04, Ubuntu 20.04, and Ubuntu 22.04
+These instructions were tested on Ubuntu 18.04, Ubuntu 20.04, and Ubuntu 22.04, and are copied from [official Gazebo Garden Install Instruction](https://gazebosim.org/docs/garden/install_ubuntu).
```

Comment:
Were these instructions tested on Ubuntu 18.04, Ubuntu 20.04 ? Won't we have the same incompatibility between the `gz` command of gazebo-garden and gazebo11?

Reply:
At one point things worked on 18.04 and 20.04 with the x500 model, but I'm not sure that is the case anymore.

Reply:
@hamishwillee 
I just tried on ubuntu 20.04 and installing `gz-garden` automatically removes gazebo classic. I suggest that we specify that gazebo classic and gz-garden are mutually exclusive in any platform